adding final dropdowns for feature flag dev hub menu (#7156)
harness/developer-hub: docs/feature-flags/get-started/feature-flag-best-practices.md

### Rollout rules

> Create a set of rules with your team to ensure your rollouts run smoothly. There are a number of Harness Features you can utilize to help with this, such as using [approval steps in pipelines](/docs/feature-flags/use-ff/ff-build-pipeline/build-feature-flag-pipeline) or using the [Harness Policy Engine](/docs/feature-flags/troubleshoot-ff/harness-policy-engine). When creating roll out rules, it's important to consider:

Reviewer: ```suggestion
Create a set of rules with your team to ensure your rollouts run smoothly. There are a number of Harness Features you can utilize to help with this, such as using [approval steps in pipelines](/docs/feature-flags/use-ff/ff-build-pipeline/build-feature-flag-pipeline) or using the [Harness Policy Engine](/docs/feature-flags/troubleshoot-ff/harness-policy-engine). When creating roll-out rules, it's important to consider:
```

However, over time, you'll want to phase out this flag as the features become permanent in production. This one flag alone might flip fifty others all on its own! You can imagine how it would be hard to find this flag to flip the features on, and then to clean it up. Here, it might make more sense to call the flag `Privacy_Features_Europe`.

> In the end, the point is to make individual feature flags easier to find, especially so if non-engineering users will be making use of them. With Harness, you can even enforce naming conventions with the use of [global governance policies for feature flags](/docs/feature-flags/troubleshoot-ff/harness-policy-engine).

Reviewer: ```suggestion
Your end goal is to make individual feature flags easier to find, espcially for non-engineering users. With Harness, you can even enforce naming conventions with the use of [global governance policies for feature flags](/docs/feature-flags/troubleshoot-ff/harness-policy-engine).
```
Optional sugs.
